spec/fixtures/entries_multivalue.obj

```diff
@@ -0,0 +1,4 @@
+[Iu:Net::LDAP::Entry�dn: uid=zach,ou=users,dc=puppetlabs,dc=com
```

Comment:
I'm seeing a weird character here and in the first line of `spec/fixtures/entries_objectClass.obj` - encoding issue?